docs/dev/setup/ide/eclipse.md

```diff
@@ -1,32 +1,53 @@
-  - [Gradle integration](https://marketplace.eclipse.org/content/buildship-gradle-integration)
-
-Useful:
-  - [EGit in Eclipse](http://www.eclipse.org/egit/) - with [tutorial](http://www.vogella.com/tutorials/EclipseGit/article.html)
-
-Lombok:
-Install Plugin ([reference](https://projectlombok.org/downloads/lombok.jar)):
-- Download 1.16.20 (Feb 24, 2018; check gradle build file for most recent) from: https://projectlombok.org/all-versions
-- Execute it: `java -jar lombok.jar`
-- If the installer does not work, there is a manual install option: https://groups.google.com/forum/#!topic/project-lombok/3rVS0eXVl5U
-- On version upgrades these steps may need to be repeated
-Enable annotation processing: 
-- https://stackoverflow.com/questions/43404891/how-to-configure-java-annotation-processors-in-eclipse
-
-  
-Checkstyle:
+# *Eclipse* Setup
+
+
+##  Plugins:
+  - ***Buildship Gradle Integration***
+     1. [Install Plugin](https://marketplace.eclipse.org/content/buildship-gradle-integration)
+  - ***CheckStyle***
+     1. [Install Plugin](http://eclipse-cs.sourceforge.net)
+     1. Follow: https://checkstyle.org/eclipse-cs/#!/project-setup
+     1. After step 3 "Activate Checkstyle for your project", select the TripleA configuration file instead of "Sun Checks"
+  - ***Lombok*** 
+    1. Download [jar file](https://projectlombok.org/downloads/lombok.jar) and execute it: `java -jar lombok.jar`
+       - [manual install](https://groups.google.com/forum/#!topic/project-lombok/3rVS0eXVl5U)
+         if that does not work
+       - Version to use: 1.16.20 
+    2. Enable annotation processing: [how-to-configure-java-annotation-processors-in-eclipse
+      ](https://stackoverflow.com/questions/43404891/how-to-configure-java-annotation-processors-in-eclipse)
+
+## Formatter
+1. *Project > Java Code Style > Formatter*
+1. Select the Triplea formatter file: [triplea/.eclipse/format/triplea_java_eclipse_format_style.xml
+   ](https://github.com/triplea-game/triplea/blob/master/.eclipse/format/triplea_java_eclipse_format_style.xml)
+    ![screenshot from 2019-01-10 15-04-53](https://user-images.githubusercontent.com/12397753/51002741-1e4fea00-14e9-11e9-86b6-4314abfb1fcd.png)
+
+## Cleanup
+
+1. *Project > Java Code Style > Cleanup*
+1. Select the Triplea cleanup file: [triplea/.eclipse/format/triplea_java_eclipse_format_style.xml
+   ](https://github.com/triplea-game/triplea/blob/master/.eclipse/format/triplea_java_eclipse_format_style.xml)
+
+![screenshot from 2019-01-10 15-07-46
+  ](https://user-images.githubusercontent.com/12397753/51002909-acc46b80-14e9-11e9-8a49-80281769f81a.png)
+
+
+## Import Order
+
+
+1. *Project > Java Code Style > Organize Imports*
+1. Click 'Import' and select the import file: [triplea/.eclipse/format/triplea.importorder
+](https://github.com/triplea-game/triplea/blob/master/.eclipse/format/triplea.importorder)
+
+Should look like this when configured: ![screenshot from 2019-01-10 15-10-27
```

Comment:
Probably want the image on a separate line so it's not pushed to the right by the text.

Reply:
Good suggestion, done: 164ac2864